Civi/Funding/Api4/Action/FundingCase/GetFieldsAction.php

```diff
@@ -110,6 +112,27 @@ protected function getRecords(): array {
         // Without sql renderer the query would fail. The actual value is fetched afterward.
         'sql_renderer' => fn () => '(SELECT NULL)',
       ],
+      [
+        'name' => 'application_process_progress',
+        'title' => E::ts('Review Progress'),
+        'description' => E::ts('The progress of application review.'),
+        'type' => 'Extra',
+        'data_type' => 'Integer',
+        'readonly' => TRUE,
+        'nullable' => FALSE,
```

Comment:
Currently it can be `NULL` if there's no application process in the funding case. The alternative option would be to change the query to return `100`. I think `NULL` would make more sense. WDYT?

Reply:
i am indifferent. will set `'nullable' => TRUE`